plasmapy/physics/transport.py

```diff
@@ -107,98 +168,745 @@ def Coulomb_logarithm(T, n_e, particles, V=None):
+        # classical Landau-Spitzer fails for argument of Coulomb logarithm
+        # Lambda < 0, therefore a clamp is placed at Lambda_min = 2
+        ln_Lambda = np.log(bmax / bmin)
+        if ln_Lambda < 2:
+            ln_Lambda = 2
+    elif method == "GMS-4":
+        # 4th method listed in Table 1 of reference [3]
+        # Spitzer-like extension to Coulomb logarithm by noting that
+        # Coulomb collisions take hyperbolic trajectories. Removes
+        # divergence for small bmin issue in classical Landau-Spitzer
+        # approach, so bmin can be zero. Also doesn't breakdown as
+        # Lambda < 0 is now impossible, even when coupling parameter is large.
+        ln_Lambda = 0.5 * np.log(1 + bmax ** 2 / bmin ** 2)
+    elif method == "GMS-5":
+        # 5th method listed in Table 1 of reference [3]
+        # Similar to GMS-4, but setting bmin as distance of closest approach
+        # and bmax interpolated between Debye length and ion sphere radius.
+        # Lambda < 0 impossible.
+        ln_Lambda = 0.5 * np.log(1 + bmax ** 2 / bmin ** 2)
+    elif method == "GMS-6":
+        # 6th method listed in Table 1 of reference [3]
+        # Similar to GMS-4 and GMS-5, but using interpolation methods
+        # for both bmin and bmax.
+        # Lambda < 0 impossible.
+        ln_Lambda = 0.5 * np.log(1 + bmax ** 2 / bmin ** 2)
+    # applying dimensionless units
+    ln_Lambda = ln_Lambda.to(u.dimensionless_unscaled).value
+    return ln_Lambda
 
-    if not isinstance(particles, (list, tuple)) or len(particles) != 2:
-        raise ValueError("The third input of Coulomb_logarithm must be a list "
-                         "or tuple containing representations of two charged "
-                         "particles.")
+@check_quantity({"T": {"units": u.K, "can_be_negative": False},
+                 "n_e": {"units": u.m**-3}
+                 })
+def b_perp(T,
+           particles,
+           V=None):
+    """
+    Distance of closest approach for a 90 degree Coulomb collision
+    """
+    # boiler plate checks
+    T, masses, charges, reduced_mass, V = boilerPlate(T=T,
+                                                      particles=particles,
+                                                      V=V)
+    # Corresponds to a deflection of 90 degrees, which is valid when
+    # classical effects dominate.
+    # !!!Note: an average ionization parameter will have to be
+    # included here in the future
+    bPerp = (charges[0] * charges[1] / 
+             (4 * pi * eps0 * reduced_mass * V ** 2))
+    return bPerp
+    
+
+@check_quantity({"T": {"units": u.K, "can_be_negative": False},
+                 "n_e": {"units": u.m**-3}
+                 })
+def impact_parameter(T,
+                     n_e,
+                     particles,
+                     V=None,
+                     method="classical"):
+    r"""Impact parameter for classical Coulomb collision
+    inputs:
+    temperature (eV)
+    ionization (dimless)
```

Comment:
These notes, on further thought, are nice in that they'll make it easier to update `Parameters` later.

Reply:
Oh! I was just keeping these until I fixed the equations to be in SI format, as many of my equations use eV and cm-3!